### DISCUSSION TITLE
When will support for mongo be available？

### DISCUSSION THREAD

**Discussion by susoooon** [zilliztech/GPTCache#417] [General]
I need an approximate time frame so that I can decide whether to implement it myself

**Comment by susoooon** [comment]
Thank u

**Comment by SimFG** [comment]
time frame? what's meaning? When the cache hits, do you need to get the time of the cache insertion?

And mongo is as a scale store?

**Reply by susoooon** [reply]
Time of the cache insertion,I need it!
As for MongoDB, it is a NoSQL document database that is often used as a scalable storage solution for applications that require large amounts of data to be stored and accessed quickly.

**Reply by SimFG** [reply]
Got it, i will consider about the time of cache insertion in the next version. And about the mongodb, i will add it to our plan

**Comment by SimFG** [comment]
@susoooon the mongo will be supported in the next version 0.1.34  #454
